feat(ars548): use new UDP socket

mojomex opened the pull request:
## PR Type

- Improvement

## Related Links

- #231 -- introduces the new UDP socket (receive-only)
- #272 -- provides the send functionality needed for this PR

## Description

Introduce the new UDP socket to the ARS548 hardware interface. This 
* allows for accurate ingress timestamping
* will allow us to phase out transport_drivers in the future (#267)

## Review Procedure

Test with a real sensor if all data is sent/received as expected.

## Remarks

<!-- Write remarks as you like if you need them. -->

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

knzo25 commented:
@mojomex 
Tested on the bench by launching the radars.launch independently
C

knzo25 commented:
@mojomex 
Can you tell me what changed since the last time I reviewed?
Kind of lost track with the force pushes

mojomex commented:
@knzo25 Nothing really, just rebased onto the newest main. It seems I lost a line on the way, so I added it back in. Will be testing later today, so if transmission still works this is good to go.

knzo25 commented:
Well, I will leave this approved then
